test/function_spec.js

```diff
@@ -29,6 +29,10 @@ describe('function node', function() {
         helper.unload();
     });
 
+    after(function(done) {
```

Comment:
We should fix this in the core test code if it is causing problems

Reply:
Agree